buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -22,7 +22,7 @@ object Versions {
     const val okhttp = "4.11.0"
 
     const val firebaseAdmin = "9.1.0"
-    const val radarSchemas = "0.8.7-SNAPSHOT"
+    const val radarSchemas = "0.8.6"
```

Comment:
Should we update this to the latest one? [0.8.7-hotfix](https://github.com/RADAR-base/RADAR-Schemas/releases/tag/v0.8.7-hotfix)

Reply:
`0.8.7-hotfix` doesn't work for me. I revert it to the working version.

Reply:
Oh ok I see, I didn't realise it didn't have the time and only the date.